Makefile

```diff
@@ -156,11 +163,11 @@ jenkins-setup-dev: jenkins-setup
 .PHONY: jenkins-setup-dev
 
 jenkins-config-updater-build:
```

Comment:
Do we want separate targets for these? There is already a `prow-plugins` target for deploying plugins and we should get one for builds too.

Reply:
Ack, we can collapse this